### PR TITLE
Added Generic Paramters for getModelSource and TargetConnectins #155

### PR DESCRIPTION
**Pull request by azoitl**
https://github.com/eclipse-gef/gef-classic/issues/155

### CODE REVIEW

File: org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java [hunk 1]
```diff
@@ -498,8 +498,8 @@ protected IFigure getLayer(Object layer) {
 	 * @return the List of model source connections
 	 */
 	@SuppressWarnings("static-method")
-	protected List getModelSourceConnections() {
-		return Collections.EMPTY_LIST;
+	protected List<? extends Object> getModelSourceConnections() {
```
Comment (ptziegler) [inline org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java:501]: I don't think it's necessary to explicitly extend Object.
```suggestion
	protected List<?> getModelSourceConnections() {
```

File: org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java [hunk 2]
```diff
@@ -514,8 +514,8 @@ protected List getModelSourceConnections() {
 	 * @return the List of model target connections
 	 */
 	@SuppressWarnings("static-method")
-	protected List getModelTargetConnections() {
-		return Collections.EMPTY_LIST;
+	protected List<? extends Object> getModelTargetConnections() {
```
Comment (ptziegler) [inline org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java:517]: Same here.
```suggestion
	protected List<?> getModelTargetConnections() {
```

File: org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java (hunk 1 above)
Comment (azoitl) [reply org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java]: Yes. Fixed.

File: org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java (hunk 2 above)
Comment (azoitl) [reply org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java]: Yes. Fixed.

**Review by ptziegler — APPROVED** [review]

**Pull request merged**